### PR TITLE
Add telos and aurora chains to Symbiosis protocol

### PR DESCRIPTION
**Pull request by allush**
**NOTE**

1. The protocol is usually listed within 24 hours of merging the PR
2. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
3. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
4. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
5. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/symbiosis-finance exports TVL: 
        
 
 
    ethereum                  1.12 M
    avax                      731.69 k
    polygon                   932.59 k
    bsc                       1.39 M
    telos                     100.26 k
    boba                      393.32 k
    aurora                    4.61 k
    
    total                    4.67 M

File: projects/helper/portedTokens.js [hunk 1]
```diff
@@ -2061,6 +2075,7 @@ const chainTransforms = {
   reichain: transformReichainAddress,
   nova: transformNovachainAddress,
   canto: transformCantoAddress,
+  telos: transformTelosAddress(),
```
Comment (g1nt0ki) [inline projects/helper/portedTokens.js:2078]: think this should be `telos: transformTelosAddress,`

Comment (allush) [reply projects/helper/portedTokens.js]: sorry, my mistake.

fixed

**Comment by llamatester** [comment]
The adapter at projects/symbiosis-finance exports TVL: 
        
 
 
    ethereum                  1.12 M
    avax                      730.53 k
    polygon                   932.10 k
    telos                     99.99 k
    bsc                       1.38 M
    aurora                    4.60 k
    boba                      392.86 k
    
    total                    4.66 M

**Comment by g1nt0ki** [comment]
thanks for the PR :)

**Pull request merged**